internal/pally/scoped_test.go

```diff
@@ -0,0 +1,90 @@
+package pally_test
```

Comment:
This example is similar to what I expect us to do when constructing dispatchers. We can also safely thread the pally registry through to any sub-objects that need to own their own metrics.